server/bin/pbench-index.py

```diff
@@ -517,6 +418,131 @@ def main(options, name):
     return res
 
 
+def collect_tb(idxctx, sd_obj, ARCHIVE_rp, qdir):
+    # find -L $ARCHIVE/*/$linksrc -name '*.tar.xz' -printf "%s\t%p\n" 2>/dev/null | sort -n > $list
+    tarballs = []
+    try:
+        tb_glob = os.path.join(ARCHIVE_rp, "*", sd_obj.linksrc, "*.tar.xz")
+        for tb in glob.iglob(tb_glob):
```

Comment:
I guess we can replace `os.path` with `pathlib.Path` to be more consistent 
here something like
```
tb_glob = pathlib.Path(ARCHIVE_rp, "*", sd_obj.linksrc)
for tb in tb_glob.rglob("*.tar.xz")
```

Reply:
it seems Path.glob doesn't deal with "*" in between their path due to which it is creating error "(ARCHIVE_rp, "*", sd_obj.linksrc)" like here

Reply:
Yes, not sure you can use `pathlib` here effectively.